widget/playground/src/constants/presets.ts

```diff
@@ -42,20 +42,41 @@ export const PRESETS: {
       neutral: '#E6E6E6',
       background: '#FDFDFD',
       foreground: '#010101',
+      info: '#5BABFF',
     },
   },
   {
```

Comment:
When I switch between these two presets in dark theme, it seems fine. But in light theme, nothing changes. (custom colors item e.g. for neutral doesn't change)
![Recording 2023-11-15 at 11 28 19](https://github.com/rango-exchange/rango-client/assets/99180934/f7a1134d-73c1-43fb-94cd-4a22724acf41)

Reply:
These two themes are related to Rango and they are different only in the dark theme